Applications/SystemMonitor/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(SOURCES
     ProcessFileDescriptorMapWidget.cpp
     ProcessMemoryMapWidget.cpp
     ProcessModel.cpp
-    ProcessStacksWidget.cpp
+    ProcessStackWidget.cpp
```

Comment:
Seems like this should be called `ThreadStackWidget` now, no? :)

Reply:
[Indeed!](https://www.youtube.com/watch?v=bqxJdl5Wbwc)